### PR TITLE
Fix api endpoints

### PR DESCRIPTION
**Pull request by Phaedra144**

### CODE REVIEW

File: app/src/main/java/com/greenfox/peridot/peridot_coz_android/api/MockService.java [hunk 1]
```diff
@@ -36,6 +37,25 @@
...
+    }
+
+    @Override
+    public Call<UsersResponse> getUsers() {
+        return new MockCall<UsersResponse>() {
+            @Override
+            public void enqueue(Callback<UsersResponse> callback) {
+                users.add(new User("Balint"));
+                users.add(new User("Bea"));
+                users.add(new User("Szilvi"));
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/peridot/peridot_coz_android/api/MockService.java]: @Phaedra144 move these to the constructor so it wont add them everytime this gets called

File: app/src/main/java/com/greenfox/peridot/peridot_coz_android/api/MockService.java [hunk 2]
```diff
@@ -76,14 +96,22 @@ public void enqueue(Callback<Troop> callback) {
...
+            }
+        };
+    }
+
+    @Override
+    public Call<Troop> upgradeTroop(@Body final Troop troop) {
+        return new MockCall<Troop>() {
+            @Override
+            public void enqueue(Callback<Troop> callback) {
+                Response<Troop> r = Response.success(new Troop(10, 2, 1));
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/peridot/peridot_coz_android/api/MockService.java]: @Phaedra144 use the `troop` passed as param, update it and send that back

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/greenfox/peridot/peridot_coz_android/api/MockService.java (hunk 1 above)
Comment (Phaedra144) [reply app/src/main/java/com/greenfox/peridot/peridot_coz_android/api/MockService.java]: Done :)

File: app/src/main/java/com/greenfox/peridot/peridot_coz_android/api/MockService.java (hunk 2 above)
Comment (Phaedra144) [reply app/src/main/java/com/greenfox/peridot/peridot_coz_android/api/MockService.java]: I hope it's done :D

Comment (Phaedra144) [reply app/src/main/java/com/greenfox/peridot/peridot_coz_android/api/MockService.java]: committed corrections

**Review by adamgyulavari — APPROVED** [review]

**Pull request merged**